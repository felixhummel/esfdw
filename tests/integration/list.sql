SELECT
  my_list,
  my_array,
  different_sep,
  list_with_objects,
  list_with_objects__b,
  list_with_objects_as_json
FROM hello
;
