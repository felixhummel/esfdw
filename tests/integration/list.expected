two|another|{two,another}|two,another|{"a": 1, "b": "doc1-b1"},{"a": 2, "b": "doc1-b2"},{"something": {"completely": "different"}}|doc1-b1,doc1-b2|[{"a": 1, "b": "doc1-b1"}, {"a": 2, "b": "doc1-b2"}, {"something": {"completely": "different"}}]
one|another|{one,another}|one,another|{"a": 1, "b": "doc1-b1"},{"a": 2, "b": "doc1-b2"}|doc1-b1,doc1-b2|[{"a": 1, "b": "doc1-b1"}, {"a": 2, "b": "doc1-b2"}]
